clang-tools-extra/clangd/unittests/ParsedASTTests.cpp

```diff
@@ -768,6 +764,35 @@ TEST(ParsedASTTest, GracefulFailureOnAssemblyFile) {
       << "Should not try to build AST for assembly source file";
 }
 
+TEST(ParsedASTTest, PreambleWithDifferentTarget) {
+  constexpr std::string_view kPreambleTarget = "x86_64";
```

Comment:
I'm not sure if we'll hit this in practice, but this test will fail if the specified targets are disabled during builds.
We might need to guard it against wrong build configurations.

I'd wait until some buildbot fails, though, maybe we never actually run Clangd tests when the set of targets is small.

Reply:
yeah happy to wait and see if some build bots fail